src/libs/utillib/options_base.cpp

```diff
@@ -405,7 +405,7 @@ namespace qblocks {
         os << notes();
         if (!isReadme) {
             os << bBlue << "  Powered by QBlocks";
-            os << (isTestMode() ? "" : " (" + getVersionStr(true) + ")") << "\n" << cOff;
+            os << (isTestMode() ? "" : " (" + getVersionStr(true) + ")") << " [" << GIT_BRANCH << "@" << GIT_COMMIT_HASH << "]\n" << cOff;
```

Comment:
This is great, but do me a favor? Can you (1) enter the command `getBlock --latest`. You should see the Parity version string:

```
Parity-Ethereum//v2.3.4-beta-0e95db11d-20190220/x86_64-macos/rustc1.32.0
```

Let's make our string look like this up to but NOT including the operating system. So,

```
GHC-TrueBlocks//v2.3.4-alpha-COMMIT-YYYYMMDD
```

C++ has built-in __YEAR__ defines I think.

Also, did you think about putting this inside of getVersionStr? In that way, the change would take place everywhere. I think that would be okay.

Reply:
Done. #47.

Reply:
I might like to include the branch name too, but I'll defer. The branch name can sort of be inferred by the commit hash.

Reply:
Do you want the date the user built the software here, or the date of the commit?